### DISCUSSION TITLE
[Insights] 🥇🥈🥉 List of Top Discussions participants

### DISCUSSION THREAD

**Discussion by cpholguera** [community/community#10670] [Discussions]
As part of the new Discussions dashboard on the insights tab, it would be great to have a list of the top participants, similar to this one:

https://github.com/OWASP/owasp-mstg/graphs/contributors

So that we know who exactly is more engaged. We'd like to encourage people to participate and with this we could regularly give an announcement about e.g. the Top 3 participants/contributors on our Discussions.

**Comment by marcvanzee** [comment]
+1
